src/auxiliary/auxiliary.jl

```diff
@@ -278,5 +278,16 @@ macro trixi_timeit(timer_output, label, expr)
   end
 end
 
+function split_filename(filename)
+
+  i = findlast(==('.'),filename)
+
+  if isnothing(i)
+    return filename
+  end
+
+  return filename[1:i-1],filename[i+1:end]
+
+end
```

Comment:
I am on my phone right now, so I can't check it manually. But I am pretty sure there is a function like this in `Base` - split extension or something like this?

Reply:
Yes: https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.splitext

Reply:
Removed this function and replaced it with `splitext`. Thanks for the hint!